File: scratch/find_tabs.js
const fs = require('fs');
const path = require('path');

const filepath = "C:\\Users\\박찬규\\.gemini\\antigravity\\scratch\\kfcman-link\\public\\js\\app.js";
const content = fs.readFileSync(filepath, 'utf8');
const lines = content.split('\n');

lines.forEach((line, index) => {
  if (line.includes("switchClassroomTab") || 
      line.includes("nav-item-shortener") || 
      line.includes("nav-item-polls") || 
      line.includes("nav-item-classroom") || 
      line.includes("switchMainTab") ||
      line.includes("showPollsTab") ||
      line.includes("showShortenerTab")) {
    console.log(`Line ${index + 1}: ${line.trim()}`);
  }
});

File: scratch/find_tabs.py
import os

filepath = r"C:\Users\박찬규\.gemini\antigravity\scratch\kfcman-link\public\js\app.js"
with open(filepath, 'r', encoding='utf-8') as f:
    lines = f.readlines()

for i, line in enumerate(lines):
    if "switchClassroomTab" in line or "nav-item-shortener" in line or "nav-item-polls" in line or "nav-item-classroom" in line or "switchMainTab" in line:
        print(f"Line {i+1}: {line.strip()}")
